src/App.tsx

```diff
@@ -1,61 +1,126 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
```

Comment:
[CODE STYLE] - As per the checklist, prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. It seems that you are importing `usersFromServer` directly into the App component, which is fine, but make sure it is not imported again in child components.